### DISCUSSION TITLE
Hilfe bei Plugin Portierung und Dokumentation gesucht

### DISCUSSION THREAD

**Discussion by Schrolli91** [BOSWatch/BW3-Core#73] [General]
Hallo zusammen,

da ich im Moment nicht genügend Zeit finde, suchen wir auf diesem Weg Helfer,
welche uns dabei unterstützen wollen, die Dokumentation zu erweitern, oder
Plugins von BOSWatch 2 auf BOSWatch 3 zu protieren.

Wer daran Interesse hat, darf sich gerne melden.

Viele Grüße

**Comment by samuscherer** [comment]
Hallo zusammen,

ich würde mich mal bei Gelegenheit dransetzen und das Pushover-Plugin portieren. Ich habe Boswatch2 inzw. 3 Jahre genutzt und denke es ist an der Zeit, dass ich der Community was zurückgebe. Sollte ich die Zeit finden, kann ich mir auch vorstellen, noch weitere Plugins zu portieren, das hängt aber auch davon ab, wie gut ich mit dem ersten zurechtkomme. 

Gibt es Dinge, auf die ich direkt achten sollte?

Viele Grüße 😊

**Reply by Schrolli91** [reply]
Ich habe die Hoffnung, dass alles relevante in den Docs (https://docs.boswatch.de/develop/ModulPlugin.html) drin steht.
Falls sich Fragen ergeben sollten oder du der Meinung bist da Fehlen villt noch nützliche Infos, darf das gerne auch angepasst werden (DOCS liegen ja auch im GIT).

Ich hab ja quasi die "rosa-rote" Brille auf, was das angeht und tu' mich da etwas schwer, das aus der Sicht eines außenstehenden zu beurteilen.

**Reply by u918244** [reply]
Hallo zusammen,
gibt es hier eigentlich schon einen neuen Stand? Gerade das Pushover-Plugin wäre interessant. Ich bin leider kein Python-Programmierer und stelle daher mal dumme Fragen: 
- Ist die Übertragungsarbeit hauptsächlich ein Anpassen an zu übergebende Variablen/Werte oder ist der grundsätzliche Plugin-Aufbau anders?
- Gibt es Konflikte zwischen der Python-Version aus BOSwatch2 und BOSwatch3?
Ich bin eher "Frickler" als systematischer Programmierer - unterstütze aber gerne im Rahmen meiner Möglichkeiten. Beim Testen sowieso ;-)
